planter/planter.sh

```diff
@@ -112,8 +112,8 @@ if [ -z "${NO_PULL+set}" ]; then
 fi
 
 # run the command or echo it if dry run
-if [ -n "${DRY_RUN+set}" ]; then
-    echo "${CMD}"
-else
+if [ -z "${DRY_RUN+set}" ]; then
```

Comment:
I don't think this works quite right?

Reply:
https://stackoverflow.com/a/13864829

Reply:
This allows checking `DRY_RUN= ...` or `DRY_RUN="some-stuff" ...` vs `$DRY_RUN` is unset, which is ideal since `DRY_RUN` is just a poor flag. If it's set at all we'll treat it as a dry run, if it's not set we'll run wet.

https://github.com/kubernetes/test-infra/pull/8530#issuecomment-401574788 includes example usage of these flags